css/amp.css

```diff
@@ -287,6 +287,17 @@ template {
   display: none !important;
 }
 
+/**
+ * Provide common border-box sizing pattern to authors
+ */
+html.amp-border-box {
+  box-sizing: border-box;
+}
+html.amp-border-box *,
+html.amp-border-box *:before,
+html.amp-border-box *:after {
+  box-sizing: inherit;
```

Comment:
Great point. My thinking was that html components that are built around a different `box-sizing` model would only need to set their `box-sizing` on the parent which would inherit down. Now I'm thinking that that case in an AMP page would be rare and in that edge case the author could assign `box-sizing` on each element on that component.

Will update.